tests/specs/file.js

```diff
@@ -65,20 +65,71 @@ define(["dist/resolver"], function(Resolver) {
     });
 
 
-    describe("when addExtension", function() {
-      it("then `test/file.html` is `test/file.html", function() {
-        var fileString = File.addExtension("test/file.html", "badextension");
-        expect(fileString).to.equal("test/file.html");
+    describe("addExtension", function() {
```

Comment:
I like following the convention of starting describe with either `when` and `and`